packages/react-core/src/next/components/Wizard/types.tsx

```diff
@@ -0,0 +1,90 @@
+import React from 'react';
+import { WizardNavItemProps, WizardNavProps } from '../../../components/Wizard';
+
+/** Initially inferred from WizardStep components, these properties represent what is controllable from within WizardContext for a given step. */
+
+export interface Step {
```

Comment:
Typically if an exported  type/enum/interface is directly related to a component and we do no have the intention of using it elsewhere, we prefix it with component name. This also helps avoiding collisions.  Maybe rename it to `WizardControllableStep`?
This comment also applies to the SubStep.

Reply:
I'm not exactly sure if users will need access to `DefaultWizardFooterProps` or other types within the `types` file outside of `Step` & `SubStep`, which is why the `index` file was updated to only expose those 2; https://github.com/patternfly/patternfly-react/blob/5982c3f7cbc4e7551aa229a206168c74f57a642a/packages/react-core/src/next/components/Wizard/index.ts#L9-L11

Users would need to use an import statement like the following in order to access them:
`import { DefaultWizardFooterProps } from '@patternfly/react-core/src/next/components/Wizard/types';`.

Is our policy that if the user can access something exported in any fashion they need to be visible in our docs?

Reply:
I think If the consumer is required to use the types, it should be documented. 
The reason I mention DefaultWizardFooterProps is because it is in the prop table. If something is in the prop table I think it should be defined since it is something the consumer would need to know the definition of to use it.

![image](https://user-images.githubusercontent.com/39532947/185941046-019ab14f-474c-42d0-9904-c938ad8c14e3.png)